Solutions/LastPass/Analytic Rules/HighlySensitivePasswordAccessed.yaml

```diff
@@ -0,0 +1,45 @@
+id: b39e6482-ab7e-4817-813d-ec910b64b26e
+name: Highly Sensitive Password Accessed
+description: |
+  'This rule will monitor access to highly sensitive passwords.
+  Within the Watchlist called 'LastPass' define passwords which are deemed highly sensitive (such as password to a high privileged application).
+  When an activity is observed against such password, an incident is created.'
+severity: Medium
+requiredDataConnectors:
+  - connectorId: LastPass
+    dataTypes:
+      - LastPass_BYOC_CL
+queryFrequency: 1h
+queryPeriod: 1h
+triggerOperator: gt
+triggerThreshold: 0
+eventGroupingSettings:
+  aggregationKind: AlertPerResult
+tactics:
+  - CredentialAccess
+  - Discovery
+relevantTechniques:
+ - T1078
+ - T1134
+ - T1555
+ - T1087
+query: |
+  let watchlist = (_GetWatchlist("LastPass") | project name);
```

Comment:
Name '_GetWatchlist' does not refer to any known function. Because of that validations are failing.